tools/add_triage_signature.py

```diff
@@ -804,6 +805,32 @@ def _update_ticket(self, url, issue_key, should_update=False):
         self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class SNOMachineCidrSignature(Signature):
+    def __init__(self, jira_client):
+        super().__init__(jira_client, comment_identifying_string="h1. Invalid machine cidr")
+
+    def _update_ticket(self, url, issue_key, should_update=False):
+        url = self._logs_url_to_api(url)
+        md = get_metadata_json(url)
+
+        cluster = md['cluster']
+
+        if cluster.get("high_availability_mode") != "None":
+            return
+
+        host = cluster['hosts'][0]
+        inventory = json.loads(host['inventory'])
+        machine_cidr = ipaddress.ip_network(cluster["machine_networks"][0]["cidr"])
+        for route in inventory['routes']:
+            # currently only relevant for ipv4
+            if route.get('destination') == "0.0.0.0" \
+                    and route.get("gateway") and ipaddress.ip_address(route["gateway"]) in machine_cidr:
+                return
+
+        report = f"Machine cidr {machine_cidr} doesn't match any default route configured on the host"
```

Comment:
Why is this bad? Some extra information could be nice here. Do we have any ticket / plans / validation for this?